packages/labs/src/components/tooltip/tooltip2.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as classNames from "classnames";
+import { Modifiers as PopperModifiers } from "popper.js";
+import * as PureRender from "pure-render-decorator";
+import * as React from "react";
+
+import { Classes, IIntentProps, IProps, PopoverInteractionKind } from "@blueprintjs/core";
+
+import { Popover2 } from "../popover/popover2";
+
+export interface ITooltip2Props extends IProps, IIntentProps {
+    /**
+     * The content that will be displayed inside of the tooltip.
+     */
+    content: JSX.Element | string;
+
+    /**
+     * The amount of time in milliseconds the tooltip should remain open after the
+     * user hovers off the trigger. The timer is canceled if the user mouses over the
+     * target before it expires.
+     * @default 0
+     */
+    hoverCloseDelay?: number;
+
+    /**
+     * The amount of time in milliseconds the tooltip should wait before opening after the
+     * user hovers over the trigger. The timer is canceled if the user mouses away from the
+     * target before it expires.
+     * @default 100
+     */
+    hoverOpenDelay?: number;
+
```

Comment:
Do we still want `inheritDarkTheme`, `inline`, or (for controlled mode) `isOpen`?